src/FondOfSpryker/Yves/GoogleTagManager/Plugin/Mapper/EnhancedEcommerceProductMapper/IdProductFieldMapperPlugin.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace FondOfSpryker\Yves\GoogleTagManager\Plugin\Mapper\EnhancedEcommerceProductMapper;
+
+use Generated\Shared\Transfer\EnhancedEcommerceProductTransfer;
+use Generated\Shared\Transfer\ProductViewTransfer;
+
+class IdProductFieldMapperPlugin implements ProductFieldMapperPluginInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\ProductViewTransfer $productViewTransfer
+     * @param \Generated\Shared\Transfer\EnhancedEcommerceProductTransfer $enhancedEcommerceProductTransfer
+     *
+     * @return void
+     */
+    public function map(ProductViewTransfer $productViewTransfer, EnhancedEcommerceProductTransfer $enhancedEcommerceProductTransfer): void
+    {
+        $sku = \str_replace('ABSTRACT-', '', strtoupper($productViewTransfer->getSku()));
```

Comment:
Klappt nur bei uns und sollte über Config pflegbar sein

Reply:
@DerCalli wird in diesem Plugin nur das replacen gehandelt. Wenn ja kann man das ja über Pyz entfernen. Vielleicht sollte man das Anpassen der Sku in einer Klasse auslagern. Dann könnten sämtlich Plugins dieses nutzen. U know what i mean?